assets/js/googlesitekit/api/index.js

```diff
@@ -88,10 +88,13 @@ const siteKitRequest = async ( {
  * This method automatically handles authentication, so no credentials
  * are required to use this method.
  *
- * @param {string} type        The data to access. One of 'core' or 'modules'.
- * @param {string} identifier  The data identifier, eg. a module slug like `'search-console'`.
- * @param {string} datapoint   The endpoint to request data from.
- * @param {Object} queryParams Query params to send with the request.
+ * @param {string}  type             The data to access. One of 'core' or 'modules'.
+ * @param {string}  identifier       The data identifier, eg. a module slug like `'search-console'`.
+ * @param {string}  datapoint        The endpoint to request data from.
+ * @param {Object}  queryParams      Query params to send with the request.
```

Comment:
See above, I think for this public function it should just be called `data`. It doesn't matter to the caller whether it's query parameters or whatnot.